files_in_root.py

```diff
@@ -0,0 +1,6 @@
+def dir():
+    import os
+    this_os = os.getcwd()
```

Comment:
Looks like wrong mind of the variable name. It contains current dir, not os

Reply:
You are right, better use : this_dir. It's more understandeble.